cloud_foundry_api/datadog_checks/cloud_foundry_api/cloud_foundry_api.py

```diff
@@ -405,15 +405,22 @@ def build_dd_event(
         org_guid,
     ):
         # type: (str, str, int, str, str, str, str, str, str, str, str) -> Event
+        space_id = space_guid if space_guid else "none"
```

Comment:
Wouldn't this leave `space_guid` to be `None` instead of the string `"none"`?
Should this be:
```
space_id = space_guid = space_guid if space_guid else "none"
```
(Same as org_id below)

Reply:
`space_id` is used as a value for both `space_id:` and `space_guid:` tags - and this is exactly what was used for `space_guid` previously, so I don't see any issue (or I'm missing something :)).

Reply:
Nope, my bad. I missed that it was using the same variable for the value. LGTM